tools/mggit/tools.go

```diff
@@ -4,19 +4,18 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	"strings"
 
 	"go.einride.tech/mage-tools/mgtool"
 )
 
 func VerifyNoDiff(ctx context.Context) error {
 	cmd := mgtool.Command(ctx, "git", "status", "--porcelain")
-	var b bytes.Buffer
-	cmd.Stdout = &b
+	var status bytes.Buffer
+	cmd.Stdout = &status
```

Comment:
Should use `mgtool.Output` ?

Reply:
I didn't want to change the behaviour of the function (it returns an error and doesn't panic). 

What do you think? We can change in a follow-up PR if so.

Reply:
I was more meaning the part where we are setting stdout and then checking that status.String is empty. That part can easly just be changed to mgtool.Output? given that we have changed this anyway

Reply:
Oh yeah, i see what you mean now. Sure lets keep it